CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 ## 2.12.0 (Unreleased)
+
+FEATURES:
+* `datadog_monitor`: Add new datasource for monitors [GH-569], [GH-585]
+
+IMPROVEMENTS:
+* `datadog_synthetics_test`: Enable usage of `validatesJSONPath` operator [GH-571]
+* `datadog_synthetics_test`: Allow usage of the new assertion format [GH-571], [GH-582]
+* `datadog_synthetics_test`: Add support for `basicAuth` and `query` [GH-586]
+
+BUGFIXES:
+* `datadog_downtime`: Replace `time.LoadLocation` by tz.LoadLocation from `4d63.com/tz` package [GH-560]
+* `datadog_downtime`: Sort monitor tags to avoid unnecessary diffs [GH-540]
```

Comment:
I think we are using sets instead. This might be my fault as I am not sure if I updated the PR title.

Reply:
Right, let me fix that.